app/assets/javascripts/govuk_publishing_components/dependencies.js

```diff
@@ -5,8 +5,9 @@
 // = require jquery/dist/jquery
 // = require ./modules.js
 
-$(document).ready(function () {
+document.onreadystatechange = function () {
```

Comment:
While this is technically correct, I wonder whether it's something worth doing since we import jQuery on L1 and the modules system on L2 requires jQuery. I'm thinking we could potentially leave `dependencies.js` as is, for backwards compatibility and offering applications who still require jQuery a mean of using it from the gem. Then for the applications that no longer require jQuery, we can gradually replace the `dependencies.js` with only importing a modules system that is not based on jQuery. This would, in theory, allow a transition rather than requiring a big bang approach. Aware we have static dealing with `dependencies.js` at the moment, which may require some additional parameter being passed via slimmer. Thoughts welcome.

Reply:
The commit message on this (Remove jQuery from modules initialisation) may also be a bit misleading, as I immediately thought of `modules.js`

Reply:
Yeah, that's sensible. I've removed this commit.